shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RALStatement.g4

```diff
@@ -87,6 +87,14 @@ dropTrafficRule
     : DROP TRAFFIC RULE ifExists? ruleName (COMMA ruleName)*
     ;
 
+exportSchema
+    : EXPORT SCHEMA (CONFIGURATION | CONFIG) (FROM schemaName)? (COMMA FILE EQ filePath)?
+    ;
+
+filePath
+    : IDENTIFIER
```

Comment:
IDENTIFIER may not be suitable, as path, quotes are required, otherwise: `d:\asd` can be parsed successfully, but the result is only "d".
![image](https://user-images.githubusercontent.com/5668787/151100236-5a01d28c-c357-473f-86e5-79736df40556.png)

Reply:
Done, please review, thank you.